src/libraries/System.Drawing.Common/src/System/Drawing/GraphicsContext.cs

```diff
@@ -45,10 +45,6 @@ internal class GraphicsContext : IDisposable
         /// </summary>
         private bool _isCumulative;
 
-        private GraphicsContext()
```

Comment:
Why are you removing this?

Reply:
It's dead code that impacts nullability of fields. We've been doing this in other instances as well cc @stephentoub